cms/standard_pages/models.py

```diff
@@ -38,3 +41,92 @@ class InformationPage(BasePage):  # type: ignore[django-manager-missing]
+    featured_items = StreamField(
+        [("featured_item", RelatedContentBlock())],
+        help_text=_("Leave blank to automatically populate with child pages. Only published pages will be displayed."),
+        blank=True,
+    )
+
+    content = RichTextField(features=settings.RICH_TEXT_BASIC, blank=True)
+    related_links = StreamField([("related_link", RelatedContentBlock())], blank=True)
+
+    content_panels: ClassVar[list["Panel"]] = [
+        *BasePage.content_panels,
+        FieldPanel("summary"),
+        FieldPanel("featured_items"),
+        FieldPanel("content"),
+        FieldPanel("related_links"),
+    ]
+
+    search_fields: ClassVar[list[index.SearchField | index.AutocompleteField]] = [
+        *BasePage.search_fields,
+        index.SearchField("summary"),
+        index.SearchField("content"),
+    ]
+
+    def get_formatted_items(self, request: "HttpRequest") -> list[dict[str, str]]:
+        """Returns a formatted list of Featured items
+        that can be either children internal Pages or specified in a RelatedContentBlock
+        for use with the Design system Document list component.
+        """
+        formatted_items = []
+        if featured_items := self.featured_items:
+            for featured_item in featured_items:
+                if featured_item.value.link is None:
+                    continue
+
+                formatted_items.append(
+                    {
+                        "featured": "true",
+                        "title": {"text": featured_item.value.link["text"], "url": featured_item.value.link["url"]},
+                        "description": featured_item.value["description"],
+                    }
+                )
+
+        else:
+            for child_page in self.get_children().live().public().specific().defer_streamfields():
+                formatted_items.append(
+                    {
+                        "featured": "true",
+                        "title": {
+                            "text": getattr(child_page, "listing_title", "") or child_page.title,
+                            "url": child_page.get_url(request=request),
+                        },
+                        "description": getattr(child_page, "listing_summary", "") or getattr(child_page, "summary", ""),
+                    }
+                )
+
+        return formatted_items
+
+    def get_formatted_related_links_list(self) -> list[dict[str, str]]:
+        """Returns a formatted list of related links for both external and internal pages
+        for use with the Design System list component.
+        """
+        formatted_links = []
+
+        for related_link in self.related_links:  # pylint: disable=not-an-iterable
+            formatted_links.append(
+                {
+                    "title": related_link.value.link.get("text"),
+                    "url": related_link.value.link.get("url"),
+                }
+            )
+
+        return formatted_links
```

Comment:
Suggestion: This might be clearer as a list comprehension

Reply:
Done as part of [2dfb6f4](https://github.com/ONSdigital/dis-wagtail/pull/78/commits/2dfb6f4fb80bab1e5917fe81e27b1f1a55394914)